In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import missingno as msno
plt.style.use('ggplot')
%matplotlib inline

In [2]:
df = pd.read_csv('wholesale_customers_data.csv')
df.head()

Channel  Region  Fresh  Milk  Grocery  Frozen  Detergents_Paper  Delicassen
0        2       3  12669  9656     7561     214              2674        1338
1        2       3   7057  9810     9568    1762              3293        1776
2        2       3   6353  8808     7684    2405              3516        7844
3        1       3  13265  1196     4221    6404               507        1788
4        2       3  22615  5410     7198    3915              1777        5185

# Exploratory Data Analysis(EDA)

- shape of dataset

In [3]:
df.shape

(440, 8)

The wholesale customers dataset contains 440 rows and 8 columns

- Check columns name

In [4]:
df.columns.tolist()

['Channel',
 'Region',
 'Fresh',
 'Milk',
 'Grocery',
 'Frozen',
 'Detergents_Paper',
 'Delicassen']

- Check the columns types

In [7]:
df.dtypes.value_counts()

int64    8
dtype: int64

All features are numeric (integers 64 bit)